.github/workflows/make.yml

```
@@ -0,0 +1,15 @@
+name: make
+on: [pull_request, push]
+jobs:
+  make:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

```suggestion
      # https://github.com/actions/setup-python/releases
      - uses: actions/setup-python@v3
        with:
          # https://github.com/actions/python-versions/releases
          python-version: '3.9'
      - uses: actions/checkout@v3
```
Then below, we can install the required Python modules via `pip` instead of via APT (if it is done done by `pip install` as of `setup.cfg`?) and remove the APT commands all together?

---

The goal of this Action is to **follow the directions in the README.md**, not to create an Actions-specific workflow.